### PR TITLE
Add support for constant learning rate

### LINKED ISSUE #1185: Add support for constant learning rate
We currently do not have support for constant learning rate (in the decay phase).

### PR DESCRIPTION
**Pull request by clessig**
## Description

Add support for constant learning rate

## Issue Number

Closes #1185 

## Checklist before asking for review

-   [X] I have performed a self-review of my code
-   [ ] My changes comply with basic sanity checks:
      - I have fixed formatting issues with `./scripts/actions.sh lint`
      - I have run unit tests with `./scripts/actions.sh unit-test`
      - I have documented my code and I have updated the docstrings.
      - I have added unit tests, if relevant
-   [X] I have tried my changes with data and code:
      - I have run the integration tests with `./scripts/actions.sh integration-test`
      - (bigger changes) I have run a full training and I have written in the comment the run_id(s): `launch-slurm.py --time 60`
      - (bigger changes and experiments) I have shared a hegdedoc in the github issue with all the configurations and runs for this experiments
-   [ ] I have informed and aligned with people impacted by my change:
    - for config changes: the MatterMost channels and/or a design doc
    - for changes of dependencies: the MatterMost software development channel

### CODE REVIEW

**Comment by clessig** [comment]
Experimental results:

i7y8l91k_rytuf08d_lr

i7y8l91k_rytuf08d_val_ERA5

**Comment by Jubeku** [comment]
> More epochs in the training:
> 
> i7y8l91k_rytuf08d_hz5k7i63_br8aufmw_val_ERA5

Can you add a plot of the learning rate for the continued runs?
